extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmConfig.java

```diff
@@ -197,4 +208,90 @@ public boolean isAnyPropertySet() {
             return sql || jdbcWarnings.isPresent();
         }
     }
+
+    @ConfigGroup
+    public static class HibernateOrmConfigCache {
+
+        /**
+         * Cache configuration properties
+         */
+        @ConfigItem(name = ConfigItem.PARENT)
+        public Map<String, CacheConfig> cacheConfig;
+
+        public boolean isAnyPropertySet() {
+            return !cacheConfig.isEmpty();
+        }
+
+        public List<CacheConfigEntry> getCacheConfigurationEntries() {
```

Comment:
@gsmet In one of my earlier commits, I saw your suggestion of moving this logic somewhere else.  In the `HibernateOrmProcessor` side?